install/kubernetes/helm/istio/charts/security/templates/clusterrole.yaml

```diff
@@ -19,4 +19,25 @@ rules:
 - apiGroups: [""]
   resources: ["services"]
   verbs: ["get", "watch", "list"]
+
+{{- if $.Values.cleanUpOldCA }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Just curious, why we need this for old CA?

Reply:
This is for the service account `istio-cleanup-old-ca-service-account` be able to to perform the removal operations. The removal requires a specific set of permissions.